components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/config/SpOAuth2ExpiryTimeConfigurationTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.config;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.testutil.powermock.PowerMockIdentityBaseTest;
+
+import static org.junit.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class SpOAuth2ExpiryTimeConfigurationTest extends PowerMockIdentityBaseTest {
+
+    private static final String CONSUMER_KEY = "consumer_key";
+    private static final long TOKEN_EXPIRY_TIME = 3600L;
+
+    SpOAuth2ExpiryTimeConfiguration spOAuth2ExpiryTimeConfiguration;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        spOAuth2ExpiryTimeConfiguration = new SpOAuth2ExpiryTimeConfiguration();
+    }
+
+    @Test
+    public void testGetConsumerKey() throws Exception {
```

Comment:
Since this is just to test getter/setter, better way is to set all the values at once, and get them at once in single method. There is no much value having separate methods for each attribute.